src/olympia/reviewers/utils.py

```diff
@@ -612,6 +612,11 @@ def get_actions(self):
             can_approve_multiple = False
 
         # Definitions for all actions.
+        boilerplate_for_approve = _('Thank you for your contribution.')
+        boilerplate_for_reject = _(
+            "This add-on didn't pass review because of the following problems:\n\n1) "
+        )
```

Comment:
@wagnerand do we want to translate those ? Even if we did, that would be in the reviewer's language and not the developer, so I'm thinking no.

Reply:
That makes sense to me too. I will remove the translations of those strings, but we can also see if Andreas disagrees with that.

Reply:
We don't want to translate these. All reviewer communication is in English.